src/Symfony/Component/Lock/Tests/LockTest.php

```diff
@@ -53,8 +74,13 @@ public function testAcquireReturnsFalse()
     public function testAcquireBlocking()
     {
         $key = new Key(uniqid(__METHOD__, true));
-        $store = $this->getMockBuilder(StoreInterface::class)->getMock();
```

Comment:
Please keep the old test cases along with the updated ones. We need to make sure that implementations of the old interface still work as before – at least until 5.0.

Reply:
I think I just need to revert all of this

Reply:
No, we should also test if a class works that only implements the new interfaces.

Reply:
Update !